bundle/libraries/libraries.go

```diff
@@ -105,3 +109,7 @@ func libPath(library *compute.Library) string {
 
 	return ""
 }
+
+func isLocalLibrary(library *compute.Library) bool {
+	return library.Whl != "" || library.Jar != ""
+}
```

Comment:
This condition only holds when the rewriting/normalisation hasn't happened yet, correct?

If so, please add a comment describing the conditions under which this holds.

Reply:
It holds always, it's essentially means if it's wheel or jar we consider it local

Reply:
We should use `libPath() != ""` for this, IMO. It ~duplicates the functionality. 

It's the same function that is used to compute the matches in the first place.